scrape/manager.go

```diff
@@ -250,7 +250,7 @@ func getFqdn() (string, error) {
 
 	ips, err := net.LookupIP(hostname)
 	if err != nil {
-		return hostname, err
+		return hostname, nil
```

Comment:
Can you add a comment here?

Reply:
Sure, it was already mentioned in the function header comment FWIW.

Reply:
More that this looks wrong at first glance, and we don't want people accidentally undoing it.